src/plugins/data_source_management/public/components/data_source_selectable/data_source_selectable.tsx

```diff
@@ -198,6 +202,7 @@ export class DataSourceSelectable extends React.Component<
 
   onChange(options: SelectedDataSourceOption[]) {
     if (!this._isMounted) return;
+    options = options.filter((option) => !option.hasOwnProperty('isGroupLabel'));
```

Comment:
Let's not mutate the input param, can we create a new const for all options? Noticed a bug introduced for selectable due to addition of the group label, looks like this can fix it

Reply:
sure, created a new const. I am adding this line to fix the bug which keeps adding up "OpenSearch cluster" group label to options list,  whenever `onChange()` is triggered.  Are you talking about the same bug?

Reply:
Yes, noticed that during testing